manager.go

```diff
@@ -360,7 +360,7 @@ func (m *jobManager) sync() error {
 			j.State = prowapiv1.PendingState
 			j.Failure = ""
 
-			if j.Mode == "launch" {
+			if j.Mode == "launch" && (previous != nil && !previous.Complete) {
```

Comment:
Good catch (can you put this in its own commit)?

Reply:
Extracted into first commit.